src/libAtomVM/opcodesswitch.h

```diff
@@ -2120,10 +2120,16 @@ static bool maybe_call_native(Context *ctx, AtomString module_name, AtomString f
                 DECODE_COMPACT_TERM(timeout, code, i, next_off)
 
                 #ifdef IMPL_EXECUTE_LOOP
-                    if (!term_is_integer(timeout) && UNLIKELY(timeout != INFINITY_ATOM)) {
+                    avm_int64_t t = 0;
```

Comment:
so 0 is infinity here?

Reply:
It's only for TRACE, isn't it?

Reply:
I think I was just trying to avoid warnings with unallocated variables.